### PR TITLE
#5239 Update xss pattern detection in patterns.ts

### LINKED ISSUE #5239: jQuery `.after` is not covered by the xss pattern regex
I noticed that we use jQuery `.after` and its not covered by the xss pattern regex. (It covers `html|append|prepend|replaceWith|insertBefore|insertAfter` and `innerHTML|outerHTML`)

Note: need to ignore `test.after` usages

_Originally posted by @rrrooommmaaa in https://github.com/FlowCrypt/flowcrypt-browser/issues/1752#issuecomment-1595745172_

### PR DESCRIPTION
**Pull request by martgil**
This PR updates `patterns.ts` that includes JQuery's `.after()` usage within the codebase. This PR also updates the affected files and puts appropriate xss pattern flags for each of them. 

close #5239 

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by martgil** [comment]
ready for review. (but not yet for merging)

File: extension/js/content_scripts/webmail/gmail-element-replacer.ts [hunk 1]
```diff
@@ -783,7 +783,7 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     if (window.location.hash.startsWith('#settings')) {
       const settingsBtnContainer = $(this.sel.settingsBtnContainer);
       if (settingsBtnContainer.length && !settingsBtnContainer.find('#fc_settings_btn').length) {
-        settingsBtnContainer.children().last().before(this.factory.btnSettings('gmail')); // xss-safe-factory
+        settingsBtnContainer.children().last().before(this.factory.btnSettings('gmail'));
```
Comment (sosnovsky) [inline extension/js/content_scripts/webmail/gmail-element-replacer.ts:786]: let's add back `// xss-safe-factory` here and add `.before` to `validateTypeScriptLine`. As now we have `.after` there, but not `.before`

Comment (martgil) [reply extension/js/content_scripts/webmail/gmail-element-replacer.ts]: Yes, will do - thank you!

**Comment by martgil** [comment]
@sosnovsky ready for review again. thank you!

**Review by sosnovsky — APPROVED** [review]

**Pull request merged**